node/src/main/java/io/airlift/node/NodeConfig.java

```diff
@@ -169,4 +164,21 @@ public NodeConfig setConfigSpec(String configSpec)
         this.configSpec = configSpec;
         return this;
     }
+
+    public AddressType getAddressType()
+    {
+        return addressType;
+    }
+
+    @Config("node.internal-address-type")
```

Comment:
Why does this need to be configurable? Can't it be inferred from the address itself?